live/stage/data-stores/postgres/setup/README.md

```diff
@@ -0,0 +1,11 @@
+## Postgres Setup
+
+We cannot use SSH tunnels with the Postgres provider, and our RDS instances are only accesable from within the VPC.
```

Comment:
**Suggestion:** It's a minor thing, but it would be nice to either have a glossary or put the acronyms accompanied by their full meaning in parentheses when writing documentation. If as a new dev you don't know what a VPC is, then it's harder to understand the documentation.

Reply:
cool, I'll add links to these definitions